docs/book/advanced-guide/integrations.md

```diff
@@ -58,6 +58,7 @@ These are the third-party integrations that ZenML currently supports:
 | PyTorch                   | ✅      | Training               |                                                                                               | [pytorch](https://github.com/zenml-io/zenml/tree/main/examples/pytorch)                                                                                  |
 | PyTorch Lightning         | ✅      | Training               |                                                                                               |                                                                                                                                                          |
 | Vertex                    | ✅      | Cloud                  | Vertex AI as a ZenML step operator.                                                           | [vertex_step_operator](https://github.com/zenml-io/zenml/tree/main/examples/step_operator_remote_training)                                               |
+| Vertex                    | ✅      | Orchestrator           | Execute your ZenML pipelines using Vertex AI Pipelines                                        |
```

Comment:
@htahir1 What's your opinion on this, multiple rows with the same name to explain in more detail what each part of the integration does or one line like `Orchestrator/Step Operator`

Reply:
This will go away soon anyway when the integrations part of the new website is done, so Id say we do multiple lines for now

Reply:
But then would update line 60 to be Vertex Step Operator